python/inertialsense/logInspector/logPlotter.py

```diff
@@ -2486,11 +2341,12 @@ def altitude(self, fig=None):
                 ax[1].plot(timeGps, altGps)
             if len(timeIns) == len(altIns) and len(altIns) > 0:
                 ax[2].plot(timeIns, altIns)
-            if len(altGps) > 0:
-                ax[3].plot(timeBar, mslBar - (mslBar[0] - altGps[0]), label=("Bar %s" % self.log.serials[d]))
-                ax[3].plot(timeGps, altGps, label=("GPS %s" % self.log.serials[d]))
-            elif len(mslBar) > 0:
-                ax[3].plot(timeBar, mslBar, label=("Bar %s" % self.log.serials[d]))
+            if len(mslBar) > 0:
+                if len(altGps) > 0:
+                    ax[3].plot(timeBar, mslBar - (mslBar[0] - altGps[0]), label=("Bar %s" % self.log.serials[d]))
+                    ax[3].plot(timeGps, altGps, label=("GPS %s" % self.log.serials[d]))
+                elif len(mslBar) > 0:
+                    ax[3].plot(timeBar, mslBar, label=("Bar %s" % self.log.serials[d]))
```

Comment:
If this change was added in this PR, its because the check was failing to guard and the logInspector was crashing.  I do agree that we should sort out the code, but that isn't the purpose of this PR.  I'm going to resolve this comment, so we can merge it, but let's consider creating a story to address the mslBar, its its an issue.